commerce_coordinator/apps/frontend_app_payment/tests/test_filters.py

```diff
@@ -0,0 +1,43 @@
+""" frontend_app_payment filter Tests"""
+
+from unittest import TestCase
+from unittest.mock import patch
+
+from django.test import override_settings
+
+from commerce_coordinator.apps.frontend_app_payment.filters import DraftPaymentRequested
+
+
+class TestDraftPaymentRequestedFilter(TestCase):
+    """ A pytest Test Case for then `DraftPaymentRequested` """
+
+    @override_settings(
+        OPEN_EDX_FILTERS_CONFIG={
+            "org.edx.coordinator.frontend_app_payment.payment.draft.requested.v1": {
+                "fail_silently": False,
+                "pipeline": [
+                    'commerce_coordinator.apps.titan.pipeline.GetTitanPayment',
+                ]
+            },
+        },
+    )
```

Comment:
@pshiu were we talking about this override?

Reply:
Correct - thank you!